### ISSUE TITLE
 Represent 'inclusive wealth'

### ISSUE THREAD

**Issue by pdez90** [SDG-InterfaceOntology/sdgio#82]
Hi, this is a priority term that needs to be defined.
Data for the same can be found on http://uneplive.org/InclusiveWealth#.VrxjxBh96ig
We want to tag our datasets with the term 'inclusive wealth' and the terms 'inclusive' and 'wealth'

**Comment by pbuttigieg** [comment]
@pdez90 
Thanks! Please include some endorsed sources for definitions or contextual material for us to establish the semantic baseline.

**Comment by pbuttigieg** [comment]
A [World Bank Glossary page](http://www.worldbank.org/depweb/beyond/global/glossary.html) which may be useful.
On a high level, I would assert that wealth is the control of entities with a resource roles by a person or organisation. We would then have monetary wealth and other classes as subclasses.

**Comment by pbuttigieg** [comment]
The [Inclusive Wealth Report 2012](http://www.ihdp.unu.edu/docs/Publications/Secretariat/Reports/SDMs/IWR%20SDM%20Low%20Resolution.pdf) could be the authority here. 

Wikipedia's evaluation of IW is:
>The United Nations definition of inclusive wealth is a monetary measure which includes the sum of natural, human and physical assets.

Is there a different between the resource role and an asset role?

**Comment by pbuttigieg** [comment]
@pdez90 could you scan over our interpretations and comment on whether this is close to what you need? If not, please steer the conversation.

@mark-jensen Would you be able to create some draft classes based on the thoughts below and your own analysis?

A preliminary class for `inclusive wealth` could be a subclass of `wealth` (see below) and have a definition like "wealth which is the sum of human,  manufactured, natural, and health capital assets" 
The "adjusted inclusive wealth" measure seems important too, but this is definitely more of an information artefact.

Note that the "Key Lessons" section (p. 18) is quite important in qualifying much of the material below.

From the IWR's Summary for Decision Makers:
>How we calculate inclusive wealth The  Inclusive  Wealth  Index  (IWI)  seeks  to  measure  the  social  value  of  capital  assets  of  nations beyond manufactured capital. The index is inclusive in the sense that it accounts for other key assets as important components of the productive base of the economy, such as natural capital and human capital. The total value of capital assets – or wealth – is concretely measured by adding up the social worth of each capital type of a nation, where  the  social  (or  shadow)  prices  per  unit  of capital form act as a weight in its index of inclusive wealth

We'll definitely need `asset` with subclasses `human asset`, `natural asset`, and `rental price`. `capital asset` comes into the picture too, but its relation to the subclasses of `asset` is not clear to me just yet. The relation of assets to capital (seems like a summation over the value of assets) is needed as we then can create `human capital`, `manufactured  capital`, `natural capital`, and `health capital` to contend with the material below:

>Capital assets 
We  measure  wealth  by  studying  various  assets that can be grouped into the following four categories:  human  capital,  manufactured  capital, natural capital, and health capital (health is treated separately from human capital for a matter of exposition).

`wealth` is defined here as the total value of all `capital assets`. What is often discussed in the IWR seems to be `measured wealth`, or an information artefact which attempts to measure actual`wealth` by valuing assets. I think we should concern ourself with actual `wealth` for now, but add a comment that it can be confounded with 'measured wealth`. `social value` seems to be key to the Inclusive Wealth Index (IWI) and in defining value in general. Subclasses of `national wealth` (the total value of assets that are under a nation's control, meaning we need `nation` imported or created).


Regarding `human capital`:
>Human capital is primarily captured by measuring  the  population’s  educational  attainment and the additional compensation over the training  period,  while  the  shadow  price  per  unit  of human  capital  as  used  in  the  report  is  obtained by  computing  the  present  value  of  the  labor compensation received by workers over an entire working  life.  We  computed  shadow  prices  for 
every year within the 1990–2008 time period for each country, and used the average of this rental 
price  of  one  unit  of  human  capital  over  time  as the  representative  weight  for  entering  human 
capital into the wealth accounting framework. 

The values used in the report are more `average national human capital` over an 18 year period, and shadow prices at that. We could represent this as a subclass of `human capital` if needed. `human capital` itself is quite tricky, especially if it's unitised. Perhaps we could approach it like "the total value generated by a population of employed humans over a time period which corresponds to a working life". What that value is connected to (assets produced?) is not clear right now.

@pdez90 Could you help clarify the following:
 1. The "working life" is probably a standard time period, but is this the same for each country? That would seem odd.  
 2. It's not clear to me if the authors assume that any employed individual, regardless of how long they have left in employment, will generate value for an entire "working life".
 3. Should we differentiate between "employed" and "working"? One can be working and generating value without formal employment.

@rlwalls2008 do we have `population of humans` in PCO? If so, @mark-jensen do we import it? Would it make sense to have `population of employed humans` and `population of unemployed humans` in SDGIO (subclass of the PCO `population of humans`) to support `human capital` and similar classes? I'm in favour of that right now.

Regarding manufactured captial:
>Calculations of manufactured capital are based on  the  Perpetual  Inventory  Method  (PIM)  after 
setting an initial capital estimate. Once the initial capital  level  is  estimated,  changes  over  time  are 
derived  from  net  capital  formation  as  reported in  the  system  of  national  accounts.  Steady-state 
estimates are used for the initial calculation, thus assuming  that  the  capital-output  ratio  of  the 
economy is constant in the long-term. 

I don't find the above especially helpful, but I think this could be defined as:
"the total value of all assets (continuants with an asset role) produced by manufacturing processes"

Regarding natural capital:
>Natural  capital  assets  in  the  report  are  comprised of the following five categories: (1) forests, 
represented  by  timber  and  non-timber  forest benefits (NTFB); (2) fisheries (only for four countries);  (3)  fossil  fuels  (oil,  natural  gas,  and  coal); (4)  minerals  (bauxite,  copper,  gold,  iron,  lead, nickel, phosphate, silver, tin, and zinc; and (5) agricultural land. Total asset value is estimated by 
multiplying the physical amount available of the asset by its corresponding rental price.

ENVO can handle many of the entities noted here, SDGIO will then associate asset roles to them along with some form of value and rental price.

Regarding health capital:
>Changes in health capital are captured by extensions  or  reductions  in  life  expectancy.  Such 
changes are basically analyzed by calculating the years  of  life  remaining  of  a  given  population  in 
different  time  periods,  with  the  population  age distribution and the people’s probability of death 
being the key inputs into the model. As far as the shadow price of health capital is concerned, it is 
measured by value of a statistical life year. 

To do this deeply, we need `life expectancy` here as well as all the aspects of this calculation. 
The initial def would be "the sum of the predicted years of life remaining for each individual in a population of humans". The fact that this is tied so closely to model estimates means we should expect more complexity here. The label in the ontology needs thinking: it's really not about health directly.

Key finding 9 is quite illustrative of the need for better handling and appraisal of value in relation to human well-being:
>Estimates  of  inclusive  wealth  can  be  improved significantly  with  better  data  on  the  stocks  of 
natural, human, and social capital and their values for human well-being. 

As a supplementary note, I find Key Finding 6 (with Figure 2) quite revealing about the IWI's handling of 'sustainability' relative to one focused on managing the natural resource base needed to achieve long-term equilibrium:
>Technological innovation and/or oil capital gains outweigh  decline  in  natural  capital  and  damages 
from climate change, moving a number of countries  from  an  unsustainable  to  a  sustainable  trajectory. 

For future consideration:
>The  Adjusted  Inclusive  Wealth  Index  (IWI<sub>adj</sub>) is a corrected representation of countries’ capital assets,  factoring  in  specific  aspects  that  further affect the size of the productive base of a nation – namely carbon damages, oil capital gains, and total factor productivity. Carbon damages are estimated by multiplying total emissions by social costs,  as  derived  by  previous  studies.  Oil  capital gains are estimated at around 5 percent annually from 1990–2008. Total factor productivity measures the change in aggregate output that cannot be explained by the growth rate of observable inputs. This residual in growth accounting can be understood  as  a  proxy  variable  of  technological progress, which is hard to measure directly. 

`national productive base`, `carbon damages`, `oil capital gains`, `social cost`, `total factor productivity`, `observable economic input`, `technological progress`, `shadow price`, `[interconnected] externality'

**Comment by mark-jensen** [comment]
> mark-jensen Would you be able to create some draft classes based on the thoughts below and your own analysis?

@pbuttigieg 
Indeed, working on it now. Thanks for the excellent comments above.

**Comment by pdez90** [comment]
Hi,

Sorry it took me a while to reply.

 Goal   4       Ensure inclusive and equitable quality education and
promote lifelong learning opportunities for all

Target   4.a      Build and upgrade education facilities that are child,
disability and gender sensitive and provide safe, non-violent, inclusive
and effective learning environments for all.
4.a.1 Proportion of schools with access to: (a) electricity; (b) the
Internet for pedagogical purposes; (c) computers for pedagogical purposes;
(d) adapted infrastructure and materials for students with disabilities;
(e) single-sex basic sanitation facilities; and (f) basic handwashing
facilities (as per the Water, Sanitation and Hygiene for All (WASH)
indicator definitions)
Contacts: Mahesh Pradhan or Rui Zhang  in Environmental Education and
Training Unit (EETU) DEPI to guide you.
Agencies:  UNESCO,  UNICEF

UNESCO:
http://en.unesco.org/post2015/sites/post2015/files/UNESCOConceptNotePost2015_ENG.pdf
' Over approximately the last fifteen years, the concept of inclusive
education has evolved towards the idea that all children and young people,
despite different cultural, social and learning backgrounds, should have
equivalent learning opportunities in all kinds of schools. The focus is on
generating inclusive settings, basically implying: (a) respecting,
understanding and taking care of cultural, social and individual diversity
(education systems, schools and teachers» response to the expectations and
needs of students); (b) the provision of equal access to quality education;
and (c) close co-ordination with other social policies. This should involve
the expectations and demands of stakeholders and social actors. […]
Inclusive education can be seen as a process of strengthening the capacity
of an education system to reach out to all learners. It is, therefore, an
overall principle that should guide all educational policies and practices,
starting from the belief that education is a fundamental human right and
the foundation for a more just society. This rights-based philosophy is
outlined in international declarations, conventions and reports relevant to
inclusive education. […] In order to realise this right, the international
EFA movement has worked to make quality basic education available to all
learners. Inclusive education takes the EFA agenda forward by finding ways
of enabling schools and other centres of learning to serve all learners in
their communities. It focuses particularly on those who have traditionally
been excluded from educational opportunities. […] An inclusive education
system improves the efficiency and cost-benefit relationship of education
systems and achieving quality EFA. Inclusive education is perceived as an
optimization of the use of resources. Schools are likely to be less
expensive when all students are educated together, thus giving governments
an economic justification to move towards an inclusive education system.
(Reference Document, 48th session of the ICE, pp. 8-12)'

UNICEF: http://www.unicef.org/ceecis/education_18613.html
'recognition of the need to work towards 'schools for all' - institutions
which include everybody, celebrate differences, support learning, and
respond to individual needs''

Goal 8. Promote sustained, inclusive and sustainable economic growth, full
and productive employment and decent work for all
Agencies: WB, ILO, IRP, OECD

http://www.gsdrc.org/topic-guides/inclusive-growth/concepts-and-definitions/
has a defintion of inclusive growth

http://siteresources.worldbank.org/INTDEBTDEPT/Resources/468980-1218567884549/WhatIsInclusiveGrowth20081230.pdf
is the World Bank definition of inclusive growth Rapid and sustained
poverty reduction requires inclusive growth that allows people to
contribute to and benefit from economic growth

http://www.un.org/millenniumgoals/pdf/Think%20Pieces/12_macroeconomics.pdf
is a paper written by all the above organizations. Inclusive growth is
mentioned: 'For this reason, it is crucial that growth is inclusive – that
it provides broadly shared opportunities to accumulate productive assets
like education, that it allows people to utilize these assets in
growth-enhancing activities and to benefit from such activities, and that
it provisions for those that do not benefit directly from growth.'

Goal   9        Build resilient infrastructure, promote inclusive and
sustainable industrialization and foster innovation
Target   9.2       Promote inclusive and sustainable industrialization and,
by 2030, significantly raise industry's share of employment and gross
domestic product, in line with national circumstances, and double its share
in least developed countries
Agencies: WB, UNIDO, ITU

UNIDO defines inclusive industrialization:
https://isid.unido.org/about-isid.html :

"Inclusive" in this context means that industrial development must include
all countries and all peoples, as well as the private sector, civil society
organizations, multinational development institutions, and all parts of the
UN system, and offer equal opportunities and an equitable distribution of
the benefits of industrialization to all stakeholders.
Target   11.3        By 2030, enhance inclusive and sustainable
urbanization and capacity for participatory, integrated and sustainable
human settlement planning and management in all countries
Agencies: UNHabitat

http://unhabitat.org/wp-content/uploads/2015/04/Habitat-III-Issue-Paper-1_Inclusive-Cities-2.0.pdf:
Urbanization provides the potential for new forms of social inclusion,
including greater equality, access to services and new opportunities, and
engagement and mobilization that reflects the diversity of cities,
countries and the globe

Goal   16        Promote peaceful and inclusive societies for sustainable
development, provide access to justice for all and build effective,
accountable and inclusive institutions at all levels
Target   16.7        Ensure responsive, inclusive, participatory and
representative decision-making at all levels
Agencies:  UN Women, OHCHR, IPU. UNFPA

http://www.unwomen.org/en/what-we-do/post-2015: Peaceful and inclusive
societies uphold the rule of law and ensure equal access to justice. They
protect people from all forms of violence, including gender-based violence,
and control corruption and organized crime. Decision-making at all levels
is inclusive and responsive, and fundamental freedoms are upheld. Laws and
policies apply without discrimination. During wars or conflict, women often
have fewer resources to protect themselves and, with children, frequently
make up the majority of displaced and refugee populations. War tactics such
as sexual violence specifically target them.

Target   17.19        By 2030, build on existing initiatives to develop
measurements of progress on sustainable development that complement gross
domestic product, and support statistical capacity-building in developing
countries
In UNEP Pushpam is working on the inclusive wealth index

Hope this helps,

Lud

On Tue, Feb 16, 2016 at 6:12 AM, Mark Jensen <notifications@github.com>
wrote:

> mark-jensen Would you be able to create some draft classes based on the
> thoughts below and your own analysis?
>
> @pbuttigieg <https://github.com/pbuttigieg>
> Indeed, working on it now. Thanks for the excellent comments above.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SDG-InterfaceOntology/sdgio/issues/82#issuecomment-184494049>
> .
>



-- 
Yours Sincerely,
Priyanka deSouza

**Comment by mark-jensen** [comment]
@pdez90 and Lud: This is great help. Thanks!

**Comment by pbuttigieg** [comment]
@pdez90 
>@pdez90 and Lud: This is great help. Thanks!

Indeed, this is good material! However, it shows us that "inclusive" (cf. #83) can mean very different things. As Mark noted, this will could be represented as a composed class of all the different kinds of inclusivity we'd define at a more granular level (inclusive education, inclusive growth, inclusive wealth, etc). Until we have more of that in there, we'd have a *very* preliminary definition like 

`inclusive` =def. 'a [dependent continuant] inhering in a bearer by virtue of that bearer having a range of entities, identified by a policy or regulatory specification, as parts or participants'

The kind of dependent continuant is not so clear to me. If it's a quality, then we can't have an inclusive process (e.g. growth) unless it's just a defined class.

**Comment by pbuttigieg** [comment]
@mark-jensen could you add this class as a dependent continuant with an editor's note indicating that this has to be moved to a more meaningful superclass following further deliberation? I think my definition is a reasonable start, but will need more refinement too.

**Comment by mark-jensen** [comment]
@pbuttigieg 
> wealth is defined here as the total value of all capital assets. What is often discussed in the IWR seems to be measured wealth, or an information artefact which attempts to measure actual wealth by valuing assets. I think we should concern ourself with actual wealth for now

If `wealth` is to be "total value", then it is an information artifact. Unless _value_ is something different than a measurement. 

I think (total) `capital assets` is what `wealth` is a measurement of.

**Comment by pbuttigieg** [comment]
The value of the total capital assests of a given entity, yes. As mentioned in the natural capital thread, we need a "valuation process " which outputs a value estimate (information artefact) to make this clear.

**Comment by mark-jensen** [comment]
`wealth` = an information content entity which represents the total value of a person, population or organization's capital assets.

**Comment by pbuttigieg** [comment]
>wealth = an information content entity which represents the total value of a person, population or organization's capital assets. 

Perhaps: `wealth` = a [scalar measurement datum](http://purl.obolibrary.org/obo/IAO_0000032) which represents the total value of the assets under the control of a person, population, or organisation.

**Comment by pbuttigieg** [comment]
Maybe OBI's [`value specification`](http://purl.obolibrary.org/obo/OBI_0001933) or [`predicted value`](http://purl.obolibrary.org/obo/OBI_0001934) would be more relevant than `scalar measurement datum`, although I think aboutness is needed.

 [`predicted value`](http://purl.obolibrary.org/obo/OBI_0001934) can be very useful for classes like `inclusive wealth` #82, as these are mostly just estimates.

**Comment by pbuttigieg** [comment]
@mark-jensen could you update the class definition after https://github.com/SDG-InterfaceOntology/sdgio/issues/82#issuecomment-185874170 and https://github.com/SDG-InterfaceOntology/sdgio/issues/82#issuecomment-185881524?
Not sure if predicted value or value specification is appropriate. I'm leaning towards the former.

**Comment by mark-jensen** [comment]
@pbuttigieg Holding off on making the change to `predicted value` until we can discuss further. I don't think an estimate is always the same as a predicted value (in the OBI sense).

**Comment by pbuttigieg** [comment]
Okay. Prediction process is a bit unclear in the defense, but do rephrase the def: the way it reads juxtaposes wealth and "value of a person ".
